definitions/variable/economy/economy.yaml

```
@@ -184,241 +184,263 @@
     skip-region-aggregation: true
 
 - Price|Carbon:
-    description: Price of carbon (for regional aggregrates the weighted price of carbon
-     by subregion should be used)
+    definition: Price of carbon
+    unit: [EUR_2020/t CO2, USD_2010/t CO2]
+    region-aggregation:
+        - Price|Carbon:
+            weight: Emissions|CO2
+            drop_negative_weights: False
+        - Price|Carbon (weighted by Final Energy):
+            weight: Final Energy
+        - Price|Carbon (weighted by Electricity Final Energy):
+            weight: Final Energy|Electricity
+
+- Price|Carbon (weighted by Final Energy):
+    definition: Price of carbon
+    notes: Spatial aggregation use final energy as weighting variable. This is weight method
+     is preferable to avoid misleading aggregations when emissions are close to zero or negative.
+     This variable is computed as part of the region-processing from `Price|Carbon`.  
```

The proposed note is grammatically not correct and also hard to understand for non-experts.
```suggestion
    notes: Using CO2 emissions as weights for average prices can cause spurious results
      when emissions are negative. This variable can be used as an alternative indicator
      for carbon prices which is less biased in low-carbon scenarios.
      This variable is computed as part of the region-processing from `Price|Carbon`.
```